_integration-test/conftest.py

```diff
@@ -52,7 +52,15 @@ def configure_self_hosted_environment(request):
         with open("sentry/requirements.txt", "w") as req_file:
             req_file.write("python-ldap\n")
         os.environ["MINIMIZE_DOWNTIME"] = "1"
-        subprocess.run(["./install.sh"], check=True, capture_output=True)
+        subprocess.run(
+            ["sudo", "chown", "-R", "$(whoami):$(whoami)", "/var/lib/docker"],
```

Comment:
Nooooo don't do this. Try add current `$(whoami)` to docker group instead

Reply:
Should be `usermod -aG docker $(whoami)`

Reply:
giving this a shot